src/data.test.js

```diff
@@ -48,38 +53,40 @@ test('GET data (no password) should fail', async () => {
 });
 
 test('GET data (no password stored) should fail', async () => {
-  await fse.emptyDir(TESTDIR);
+  await fse.emptyDir(TESTDIR, err => {
+    if (err) return console.error(err)
+  });
+  //this is a hack--need to figure out why password file appears in project root.
```

Comment:
Dang. That's super weird. Doesn't have to be this PR, but eventually we should just move the password hash into the mongo database.